### PR TITLE
Real implementation of FE API's results view.

### PR DESCRIPTION
**Pull request by nworden**

### CODE REVIEW

**Comment by nworden** [comment]
Sent to Avi.

File: ui/src/pages/Results.js [hunk 1]
```diff
@@ -175,18 +185,49 @@ class SearchResultImpl extends Component {
   }
 
   render() {
+    var nameStr = this.props.result.fullNames.join(', ');
+    if (this.props.result.alternateNames.length > 0) {
+      nameStr += ' (' + this.props.result.alternateNames.join(', ') + ')';
+    }
+    const formattedTimestamp = <FormattedDate
+        value={new Date(this.props.result.timestamp + 'Z')}
```
Comment (ajhoffma) [inline ui/src/pages/Results.js]: Maybe guarantee a valid ISO 8601 timestamp from the API so you don't have to know about appending 'Z'?

Comment (nworden) [reply ui/src/pages/Results.js]: Done.

**Comment by nworden** [comment]
Not really sure why Travis hasn't started, but everything passes locally.

**Pull request merged**